controllers/Backend.php

```diff
@@ -3,7 +3,7 @@
 require_once('models/Article.php');
 require_once('models/Comment.php');
 require_once('models/Dashboard.php');
-
+require_once('models/Form.php');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)